content/300-guides/050-database/820-multi-schema.mdx

```diff
@@ -0,0 +1,101 @@
+
+</Admonition>
+
+<Admonition type="warning">
+
+Multiple database schema support is currently only available with the PostgreSQL and SQL Server connectors.
+
+</Admonition>
+
+This guide explains how to include multiple database schemas in your Prisma schema, and how to query across multiple schemas with Prisma Client.
+
+</TopBlock>
+
+## What are database schemas?
+
+Many database providers allow you to create named collections within a database that group together tables, constraints, indexes and other database features. In PostgreSQL and SQL Server these collections are known as schemas. We will refer to these collections as _database schemas_ to distinguish them from Prisma's own schema file.
+
+## How to enable the <inlinecode>multiSchema</inlinecode> preview feature
+
+Multi-schema support is currently in an early, experimental preview. To enable the `multiSchema` preview feature, you will need to add the `multiSchema` feature flag to the `previewFeatures` field of the `generator` block in your Prisma schema file:
+
+```prisma file=schema.prisma highlight=3;add
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["multiSchema"]
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+```
+
+## How to include multiple database schemas in your Prisma schema
+
+To use multiple database schemas in your Prisma schema file, add the names of your database schemas to an array in a new `schemas` field of the `datasource` block. The following example adds a `"base"` and a `"transactional"` schema:
+
+```prisma file=schema.prisma highlight=9;add
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["multiSchema"]
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+  schemas  = ["base", "transactional"]
+}
+```
+
+You do not need to change your connection string. You can still use the `schema` key of your connection string to refer to the default schema, which will be used for Prisma Client raw queries.
+
+To mark a model as belonging to a specific database schema, add a `@@schema` attribute with the name of the database schema as a parameter. In the following example the `User` model is part of the `"base"` schema, and the `Order` model is part of the `"transactional"` schema:
+
+```prisma file=schema.prisma highlight=6,14;add
+model User {
+  id     Int     @id
+  orders Order[]
+
+  @@schema("base")
+}
+
+model Order {
+  id      Int  @id
+  user    User @relation(fields: [id], references: [id])
+  user_id Int
+
+  @@schema("transactional")
+}
+```
+
+<Admonition type="warning">
+
+It is currently not allowed to have models with the same name in different schemas. This will fail to validate in your Prisma schema.
```

Comment:
```suggestion
Currently it is not allowed to have models with the same name in different database schemas; all models, across all database schemas, must have unique names. Otherwise, your Prisma schema will fail to validate.
```

Reply:
Technically these are not "database schemas", but their "`@@schema` representation in the Prisma schema". But that is terrible wording as well.

Reply:
Yes, that's why I avoided 'database schemas', but then I couldn't think of a good phrasing for describing 'schema representation within the Prisma schema'. I've tried spelling it out more pedantically but possibly just made it worse :/ 

Other ideas welcome.